init/cli/start.go

```diff
@@ -15,48 +15,36 @@
 package cli
 
 import (
-	"os"
-
 	"github.com/palantir/pkg/cli"
 	"github.com/pkg/errors"
 
 	"github.com/palantir/go-java-launcher/init/lib"
-	"github.com/palantir/go-java-launcher/launchlib"
 )
 
 func startCommand() cli.Command {
 	return cli.Command{
 		Name: "start",
 		Usage: `
-Launches the process defined by the static and custom configurations at service/bin/launcher-static.yml and
-var/conf/launcher-custom.yml. Writes its PID to var/run/service.pid and redirects its output to var/log/startup.log. If
-successful, exits 0, otherwise exits 1 and writes an error message to stderr.`,
+Ensures the service defined by the static and custom configurations at service/bin/launcher-static.yml and
+var/conf/launcher-custom.yml is running and its outputs are redirecting to var/log/startup.log and other
+var/log/$PROCESS-startup.log files. If successful, exits 0, otherwise exits 1 and writes an error message to stderr.`,
```

Comment:
Good thought - when we write things to `stderr` we're at the level of abstraction where it's not clear which `startup.log` file to write it to. So all we really can do is write to `stderr`. The client can do with that what they wish. In the main use case for this, `init.sh`, it probably makes the most sense for `init.sh` to redirect it to its own `stderr`, which the deployment infrastructure should read (since it executed `init.sh`) and deal with accordingly.

Reply:
Actually, I see us already logging things during the config parsing process to the primary startup log file, and those things may not necessarily be just about the primary process. So looks like there's precedent for doing that. I'll change the cli exit functionality to log errors to the primary startup log as well as to stderr.